kubetools/kubernetes/config/job.py

```diff
@@ -99,3 +99,8 @@ def make_job_config(
             },
         },
     }
+
+    if config.get('ttlSecondsAfterFinished'):
```

Comment:
`if 'ttlSecondsAfterFinished' in config`:

Reply:
Ack. Will change.

Reply:
Changes made and set for review.